package.json

```diff
@@ -144,6 +144,7 @@
     "quill-image-resize": "^3.0.9",
     "quill-image-resize-module": "^3.0.0",
     "quill-image-uploader": "^1.2.2",
+    "quill-mention": "^2.2.7",
```

Comment:
We don't use quill anymore, so these packages can be removed.

Reply:
ah, i'm just copying over the items that @stratusjs/angular depended on in it's package.json